backend/Events/GameStateEvents/DayStateChangeEvents.js

```diff
@@ -0,0 +1,56 @@
+const config = require('../../config.json');
+const GameStateEnum = require('../../domain/Enum/GameStateEnum');
+
+const MafiaGame = require('../../domain/MafiaGame');
+const DayStartDTO = require('../../domain/DTO/response/DayStartDTO');
+const DiscussionEndDTO = require('../../domain/DTO/response/DiscussionEndDTO');
+
+/**
+ * Event handlers and logic for `start-day`
+ * When the client invokes this event handler, the game state will change in-game, the day voting timer
+ * will start, and the server will emit `day-start` to the client, indicating that the day has begun.
+ * @param {any} io
+ * @param {any} socket
+ * @param {MafiaGame} mafiaGame
+ */
+function startDay(io, socket, mafiaGame) {
+    socket.on('start-day', () => {
+        const { roomID } = socket.player;
+        const room = mafiaGame.gameRoomsDict[roomID];
+        const TIME_TO_VOTE = config.day_total_vote_time_in_milliseconds;
+
+        room.gameState = GameStateEnum.DAY_TIME;
+
+        io.in(roomID).emit('day-start', new DayStartDTO(TIME_TO_VOTE));
+
+        setTimeout(endDiscussion(io, socket, mafiaGame), TIME_TO_VOTE);

```

Comment:
same issue as #154 this needs to be endDiscussion(io, socket, mafiaGame)

Reply:
yeaap will fix